lib/main.rb

```diff
@@ -1,35 +1,50 @@
 require 'json'
 
 class LogParser
-  attr_reader :data
-
-  def initialize file_path = "games.log"
+  def initialize(file_path = "games.log")
     begin
       @file_path = file_path
-      @data = File.readlines(@file_path, chomp: true)
+      @data = File.readlines(file_path, chomp: true)
     rescue
```

Comment:
Eu acho que usar rescue assim num é uma boa, fica mto generico e qlqr erro que der vai cair aqui, por exemplo se sem querer vc chamar um metodo pra nil, daria um `NoMethodError` e num é o intuito do seu metodo filtrar esse tipo de erro saca? Da uma olhada nas exceptions que o `readlines` levanta, e filtra somente elas.

Reply:
Vou te chamar no privado porque não consegui implementar isso.
Mas vou marcar aqui como resolvido e mergear, pra não empacar o desenvolvimento da tarefa 4.

Reply:
cara num ta certo fazer isso, se vc tem um comment, discute traz os argumento certim, mas dar como resolved sem resolver e sem falar o pq, ta errado. Isso é mal visto na comunidade opensource!

No caso ai, eu me expressei errado tbm, eu quis dizer pra vc n dar rescue geralzao assim, pq vc pode ter algum comportamento que te devolve a string que vc colocou no rescue, mas na verdade a treta é outra coisa. Esse [stackoverflow](https://stackoverflow.com/a/10048406/12934208) da uma explicada melhor
```suggestion
    rescue StandardError
```